templates/wordpress-woocommerce/files/.environment

```diff
@@ -0,0 +1,23 @@
+export DB_NAME=$(echo $PLATFORM_RELATIONSHIPS | base64 --decode | jq -r ".database[0].path")
+export DB_HOST=$(echo $PLATFORM_RELATIONSHIPS | base64 --decode | jq -r ".database[0].host")
+export DB_PORT=$(echo $PLATFORM_RELATIONSHIPS | base64 --decode | jq -r ".database[0].port")
+export DB_USER=$(echo $PLATFORM_RELATIONSHIPS | base64 --decode | jq -r ".database[0].username")
+export DB_PASSWORD=$(echo $PLATFORM_RELATIONSHIPS | base64 --decode | jq -r ".database[0].password")
+
+export WP_HOME=$(echo $PLATFORM_ROUTES | base64 --decode | jq -r 'to_entries[] | select(.value.primary == true) | .key')
+export WP_SITEURL="${WP_HOME}wp"
+export WP_DEBUG_LOG=/var/log/app.log
+if [ "$PLATFORM_BRANCH" != "master" ] ; then
+    export WP_ENV='development'
+else
+    export WP_ENV='production'
+fi
+
+export AUTH_KEY=$PLATFORM_PROJECT_ENTROPY
+export SECURE_AUTH_KEY=$PLATFORM_PROJECT_ENTROPY
+export LOGGED_IN_KEY=$PLATFORM_PROJECT_ENTROPY
+export NONCE_KEY=$PLATFORM_PROJECT_ENTROPY
+export AUTH_SALT=$PLATFORM_PROJECT_ENTROPY
+export SECURE_AUTH_SALT=$PLATFORM_PROJECT_ENTROPY
+export LOGGED_IN_SALT=$PLATFORM_PROJECT_ENTROPY
+export NONCE_SALT=$PLATFORM_PROJECT_ENTROPY
```

Comment:
Is the extra intermediary step of going through extra env vars worth it?  The entropy variable doesn't need JSON decoding like relationships do.  Why not just read that with `getenv()` in the config file and read it directly?  (Applies to all WP templates here.)

Reply:
WooCommerce is built on the Bedrock template. You can do both, but Bedrock's big sell is environment variables straight away.

Reply:
Ahhh... It already has the env var glue built in?  Good for them. :smile: OK, stet.